packages/core/src/components/ActionBar/ActionBar.tsx

```diff
@@ -1,23 +1,23 @@
-import { clsx } from "clsx";
 import { HvBaseProps } from "@core/types";
-import { StyledRoot } from "./ActionBar.styles";
-import actionBarClasses, { HvActionBarClasses } from "./actionBarClasses";
+import { ExtractNames } from "@core/utils";
+import { staticClasses, useClasses } from "./ActionBar.styles";
+
+export { staticClasses as actionBarClasses };
+
+export type HvActionBarClasses = ExtractNames<typeof useClasses>;
 
 export interface HvActionBarProps extends HvBaseProps {
   /** A Jss Object used to override or extend the styles applied to the component. */
   classes?: HvActionBarClasses;
```

Comment:
Nothing major because we only have one class, but to maintain backward compatibility we should use `Partial<HvActionBarClasses>`.

Reply:
Nice catch 👀